app/code/core/Mage/Adminhtml/controllers/System/AccountController.php

```diff
@@ -25,6 +25,11 @@ class Mage_Adminhtml_System_AccountController extends Mage_Adminhtml_Controller_
      */
     public const ADMIN_RESOURCE = 'system/myaccount';
 
+    protected $_publicActions = [
+        'passkeyregisterstart',
+        'passkeyregistersave',
+    ];
+
```

Comment:
@justinbeaty do you know something about these?

Reply:
Doc comment says "Array of actions which can be processed without secret key validation", so I think it's for any action that you're not logged in for. It checks for that `/key/<random_string>/` part in the URL.

 I think that since you are logged in on this page you don't need it, but maybe instead use `_forcedFormKeyActions`

Reply:
I'll retest everything (making this work was a bit cumbersome) and try to remove those 2 actions from that array